homeassistant/components/rfxtrx/cover.py

```diff
@@ -118,6 +176,26 @@ async def async_stop_cover(self, **kwargs):
         self._state = True
         self.async_write_ha_state()
 
+    async def async_open_cover_tilt(self, **kwargs):
+        """Tilt the cover up."""
+        if self._venetian_blind_mode == CONST_VENETIAN_BLIND_MODE_US:
+            await self._async_send(self._device.send_up2sec)
+        elif self._venetian_blind_mode == CONST_VENETIAN_BLIND_MODE_EU:
+            await self._async_send(self._device.send_up05sec)
+
+    async def async_close_cover_tilt(self, **kwargs):
+        """Tilt the cover down."""
+        if self._venetian_blind_mode == CONST_VENETIAN_BLIND_MODE_US:
+            await self._async_send(self._device.send_down2sec)
+        elif self._venetian_blind_mode == CONST_VENETIAN_BLIND_MODE_EU:
+            await self._async_send(self._device.send_down05sec)
+
+    async def async_stop_cover_tilt(self, **kwargs):
+        """Stop the cover tilt."""
+        await self._async_send(self._device.send_stop)
+        self._state = True
```

Comment:
I don't think stopping the tilt movement should affect the up/down state.

Reply:
See comment on support of tilt position above.   Stopping the tilt movement is implemented to do exactly what stopping the movement  does so it is ok to affect the open/close state.  Otherwise Rfy venetion blinds do not support stopping the tilt movement.  The implementation is in order to have the stop button in the tilt section do something logical instead of nothing.

Reply:
This make no sense. Stopping a cover doesn't change its open close state. It only stops it from moving.

Reply:
That's not how Somfy (Rfy) blinds work.  The Stop button on Somfy remotes does two things 1. Stop movement (if the blind is moving at that moment), and 2. If the blind is not moving - go to a pre-defined position, which by default is the closed position of the blind (with slats open).  I would guess in 90% of the use cases pressing the stop button will close the blind.  Which is probably why  the original implementation of the feature (which this PR is not changing)  is to have the Stop button set  the cover state to closed.

Reply:
Okey, then i understand. I also see that the normal stop and tilt stop uses same stop function. So okey I'm fine with this.